warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reindex/ShardReindexJob.java

```diff
@@ -139,31 +171,72 @@ private Job setupJob() throws IOException, ParseException, AccumuloException, Ta
             throw new IllegalStateException("split.work.dir and job.output.table.names must be configured");
         }
 
+        // test that each of the output table names exist
+        Properties accumuloProperties = builder.build();
+        AccumuloClient client = Accumulo.newClient().from(accumuloProperties).build();
```

Comment:
This client should be closed. I also think Accumulo recommends creating a single client object and reusing for the duration of the app now. Its supposed to be a heavier weight utility than the old Connector objects. You use another client in your `getSplitsFromMetadata()` method. Might be worth promoting the client to a class member and closing it at the end.